DuggaSys/microservices/sectionedService/setVisibleListentries_ms.php

```diff
@@ -21,6 +21,8 @@
 $courseid = getOP('cid');
 $versid = getOP('vers');
```

Comment:
$courseverse is used in the function call. I'm assuming verseid should be used. $debug needs to be declared as well